projects/plugins/jetpack/modules/carousel/swiper-bundle.css

```diff
@@ -0,0 +1,380 @@
+/* eslint-disable */
+/* DO NOT EDIT - This file is automatically generated. See readme.md */
+/**
+ * Swiper 6.7.0
+ * Most modern mobile touch slider and framework with hardware accelerated transitions
+ * https://swiperjs.com
+ *
+ * Copyright 2014-2021 Vladimir Kharlampidi
+ *
+ * Released under the MIT License
+ *
+ * Released on: June 22, 2021
+ */
+
+@font-face {
+  font-family: 'swiper-icons';
```

Comment:
Does Swiper have an option for using plain SVGs or an SVG spritemap instead? As I mentioned above, icon fonts are a bad practice for a number of reasons.

Reply:
Update for this here https://github.com/Automattic/jetpack/pull/20222

Reply:
Thank you, @glendaviesnz! 👍